i18n_generate.mll

```diff
@@ -85,23 +85,18 @@ and parse_string_2 buffer = parse
 
 let print_header fmt default_lang =
   Format.pp_print_string fmt @@
-  "let%server language =\n\
-   Eliom_reference.Volatile.eref \
-   ~scope:Eliom_common.request_scope " ^ default_lang ^ "\n\
-   \n\
-   let%server get_lang () = Eliom_reference.Volatile.get language\n\
-   \n\
-   (* For non connected users, \
-   we record the language in a session reference: *)\n\
-   let%server non_connected_language =\n\
-   Eliom_reference.Volatile.eref ~scope:Eliom_common.default_session_scope\n\
-   (if true then None else Some " ^ default_lang ^ ")\n\
-   \n\
-   let%client language = ref " ^ default_lang ^ "\n\
-   let%client non_connected_language =\n\
-   ref (if true then None else Some " ^ default_lang ^ ")\n\
-   \n\
-   let%client get_lang () = !language\n\
+  "[%%server\n\
+   let _language_ =\n\
+   Eliom_reference.Volatile.eref\n\
+   ~scope:Eba_session.user_indep_process_scope Bs_data.Us\n\
```

Comment:
This adds a dependency to EBA, is it possible to avoid it?

Also, Bs_data.Us should be `default_lang`.